download_and_create_reference_datasets/v03/write_clinvar_ht.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
```

Comment:
I made a completely separate script for `v03` so the old one (with the existing path structure and fields) is preserved.

Reply:
This a good question.  I'm not sure if we want to preserve the "run on dataproc" wrapper scripts going forwards.... though I can see a case!  I think nobody else (the open source users of this repo) would run these reference data generators right?

Reply:
I would argue we in fact *don't* want to preserve the wrapper scripts as I think they really reduce readability. Nobody but us needs to run these, so they should really work in one way, which should be the way we intend to call them

Reply:
👍 I agree!